core/src/test/scala/org/apache/spark/util/UtilsSuite.scala

```diff
@@ -461,19 +461,17 @@ class UtilsSuite extends SparkFunSuite with ResetSystemProperties with Logging {
     def assertResolves(before: String, after: String): Unit = {
       // This should test only single paths
       assume(before.split(",").length === 1)
-      // Repeated invocations of resolveURI should yield the same result
       def resolve(uri: String): String = Utils.resolveURI(uri).toString
+      assert(resolve(before) === after)
```

Comment:
Thanks for asking me. The initial change proposed looked testing several duplicated test cases and also changing existing tests to me. I am fine with the current status.